Distraction Shield/constants.js

```diff
@@ -4,6 +4,18 @@ var zeeguuExLink = "https://www.zeeguu.unibe.ch/practice/get-ex";
 /* ---------- ----- BlockedSite ----- ---------- */
 var FAVICONLINK = "https://www.google.com/s2/favicons?domain=";
 
+/* ---------- ----- Constants for statistics ----- ---------- */
+var zeeguuDomain = "zeeguu.herokuapp.com";
```

Comment:
isn't this link already in the constants? and the herokuapp domain isn't used anymore.